### PR TITLE
[Issue 5887]Fix old version download link error

### LINKED ISSUE #5887: [Docs]Old version download link error
**Describe the bug**

http://pulsar.apache.org/docs/en/2.3.0/standalone/

image

The old version has been removed https://archive.apache.org/dist/pulsar/. Only the latest two versions are kept here in https://www-eu.apache.org/dist/pulsar/.

**To Reproduce**

**Expected behavior**
Displays the correct download link

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**
We should fix this script https://github.com/apache/pulsar/blob/master/site2/website/scripts/replace.js to solve this problem.

Test

```
git clone https://github.com/apache/pulsar.git
cd pulsar/site2/website
yarn install
rm -rf build
yarn build
node scripts/replace.js
```

Check file build/pulsar/docs/en/2.3.0/standalone/index.html, search key `downloads page`:

```
<li><p>from the Pulsar <a href="/download">downloads page</a></p></li>
```

### PR DESCRIPTION
**Pull request by huangdx0726**
Fix old version download link error

Fixes #5887 

### Motivation

Old version download link error

### Modifications

Change the download link to https://archive.apache.org/dist/pulsar/

### CODE REVIEW

**Comment by tuteng** [comment]
I think this fix seems incorrect. The download link on this page http://pulsar.apache.org/docs/en/2.3.0/standalone/ should be apache-pulsar-2.3.0-bin.tar.gz instead of rpm package download link, so I think the following code should be added and then the `downloadPageUrl` function should be fixed to support the download of the old version.

```
     binaryReleaseUrl(`${v}`),
      connectorReleaseUrl(`${v}`),
      // add this function
      offloaderReleaseUrl(`${v}`),
      prestoPulsarReleaseUrl(`${latestVersion}`),
      // Fix this function for support old version
      downloadPageUrl(),
      rpmReleaseUrl(`${v}`, ""),
      rpmReleaseUrl(`${v}`, "-debuginfo"),
      rpmReleaseUrl(`${v}`, "-devel"),
      debReleaseUrl(`${v}`, ""),
      debReleaseUrl(`${v}`, "-dev"),
```

**Comment by tuteng** [comment]
run cpp tests

File: site2/website/scripts/replace.js [hunk 1]
```diff
@@ -178,13 +178,20 @@ for (v of versions) {
...
       rpmReleaseUrl(`${v}`, "-debuginfo"),
       rpmReleaseUrl(`${v}`, "-devel"),
       debReleaseUrl(`${v}`, ""),
       debReleaseUrl(`${v}`, "-dev"),
+
+      rpmDistUrl(`${v}`, ""),
+      rpmDistUrl(`${v}`, "-debuginfo"),
+      rpmDistUrl(`${v}`, "-devel"),
+      debDistUrl(`${v}`, ""),
+      debDistUrl(`${v}`, "-dev")
```
Comment (tuteng) [inline site2/website/scripts/replace.js]: You can try:

```
      `${vWithoutIncubating}`,
      `${v}`,
      binaryReleaseUrl(`${v}`),
      connectorReleaseUrl(`${v}`),
      offloaderReleaseUrl(`${v}`),
      prestoPulsarReleaseUrl(`${v}`),
      downloadPageUrl(),
      rpmDistUrl(`${v}`, ""),
      rpmDistUrl(`${v}`, "-debuginfo"),
      rpmDistUrl(`${v}`, "-devel"),
      debDistUrl(`${v}`, ""),
      debDistUrl(`${v}`, "-dev"),
      rpmDistUrl(`${v}`, ""),
      rpmDistUrl(`${v}`, "-debuginfo"),
      rpmDistUrl(`${v}`, "-devel"),
      debDistUrl(`${v}`, ""),
      debDistUrl(`${v}`, "-dev")
```
You can test the results by opening the html file `build/pulsar/docs/en/2.4.1/client-libraries-cpp/index.html` through the browser. I think this is correct.

**Comment by Jennifer88huang** [comment]
run cpp tests

**Comment by Jennifer88huang** [comment]
run cpp tests

**Comment by huangdx0726** [comment]
run cpp tests

**Comment by jiazhai** [comment]
run cpp tests

File: site2/website/scripts/replace.js (hunk 1 above)
Comment (huangdx0726) [reply site2/website/scripts/replace.js]: I'v fixed it, PTAL.

**Comment by tuteng** [comment]
retest this please

**Comment by Jennifer88huang** [comment]
retest this please

**Comment by sijie** [comment]
retest this please

**Comment by sijie** [comment]
run java8 tests

**Pull request merged**